### ISSUE TITLE
Missing kubernetes.config.load_kube_config_from_dict()

### ISSUE THREAD

**Issue by svinther** [tomplus/kubernetes_asyncio#167]
Hello,

To be able to call multiple clusters I rely on configuring the client by getting a kubeconfig from Azure AKS rest management api, and then configure kubernetes client with the function kubernetes.config.load_kube_config_from_dict()

As I would like to use this asyncio client to replace some parts, I noticed with some sadness that this function is missing.

The function header is:
```
def load_kube_config_from_dict(config_dict, context=None,
                               client_configuration=None,
                               persist_config=True,
                               temp_file_path=None):
    """Loads authentication and cluster information from config_dict file
    and stores them in kubernetes.client.configuration.

    :param config_dict: Takes the config file as a dict.
    :param context: set the active context. If is set to None, current_context
        from config file will be used.
    :param client_configuration: The kubernetes.client.Configuration to
        set configs to.
    :param persist_config: If True, config file will be updated when changed
        (e.g GCP token refresh).
    :param temp_file_path: store temp files path.
    """
```

**Comment by tomplus** [comment]
I'll add it soon.

**Comment by svinther** [comment]
Great news, I will look out for it. Thanks @tomplus

**Comment by tomplus** [comment]
@svinther The function is available in v18.20.1.
